contextproject-ms3/src/main/java/contextproject/models/BeatGrid.java

```diff
@@ -0,0 +1,66 @@
+package contextproject.models;
+
+public class BeatGrid {
+
+  private long length;
+  private float bpm;
+  private long firstBeat;
```

Comment:
Legnth and bpm are not needed to be local variables, since you only use them in your constructor.